Description: Disable doctests
  Doctests and Template Haskell are incompatible. The following ghc
  bug seems related:
    https://ghc.haskell.org/trac/ghc/ticket/8025
  Re-enable doctests once this has been fixed.
Author: Ilias Tsitsimpis <iliastsi@debian.org>

Index: b/trifecta.cabal
===================================================================
--- a/trifecta.cabal
+++ b/trifecta.cabal
@@ -89,6 +89,7 @@ library
     build-depends: fail == 4.9.*
 
 test-suite doctests
+  buildable: false
   type:             exitcode-stdio-1.0
   main-is:          doctests.hs
   ghc-options:      -Wall -threaded
